### ISSUE TITLE
Autosave no longer working with new update

### ISSUE THREAD

**Issue by Michaela531** [al-caughey/Google-Meet-Attendance#73]
Yesterday I noticed the autosave was not working any longer. I played around with settings and noticed the auto-save feature was set to "no". I changed it to "yes" and the auto-save feature is still not working.

**Comment by Sherry49** [comment]
Yes!! I am so bummed, this extension was lifesaver because when I logged off each meet the HTML with the green and pink bars would automatically save and pop up.  Now it's gone, and no matter what I do in the settings I can't get it to work again.  I get the CSV gets saved, but thats not the one I want.  :(

**Comment by Michaela531** [comment]
Autosave of the CSV file isn't even working for me - that's what I prefer, but I can't see my attendance saved in any fashion.

**Comment by Eagle-Man** [comment]
Should be fixed with 1.0.2?  Also, make sure you end the Meet properly with the button and not by closing the tab/window.  That may come later but is not in there now.

**Comment by lmuch2020** [comment]
I have the update from over the weekend and the AutoSave to html is not working. It shows that html version is available but when I click it, nothing happens.  It worked flawlessly before this update.  Our district is going virtual now and it sure would be helpful to have the data.  Thanks!

**Comment by Michaela531** [comment]
I removed the extension from Chrome and added it back and that seemed to fix my attendance auto-save issue! (At least for now...)

**Comment by AlCaughey** [comment]
I expect that you have v1.0.3 now... I fixed the issue and posted the
update last night

**Comment by lmuch2020** [comment]
I tried again today and get the same error.  Screenshot attached showing it
says it has autosaved the html file but it is nowhere.
*Lori A. Much*
Spanish Teacher
Booster Club President
Sheboygan North High School
920-459-3600

*Live each day to the fullest and do not take anything for granted.*

**Comment by lmuch2020** [comment]
I tried again and it says that it has autosaved the html file but it is nowhere.
<img width="476" alt="Screen Shot 2020-10-14 at 2 15 49 PM" src="https://user-images.githubusercontent.com/72869333/96035033-09c02480-0e28-11eb-9ce3-f9c3c3f7497a.png">

**Comment by Eagle-Man** [comment]
Have you checked in chrome://downloads to see if the browser received anything?  That can also allow you to open the folder where it was saved.

**Comment by lmuch2020** [comment]
I have looked there......before you did the update, the html files show there just fine.  Since then, only the csv files do.

**Comment by Eagle-Man** [comment]
I'm not sure what to say.  No one else has reported this kind of problem.  It sounds like the information is sent to the browser, but the browser (not the extension) is not doing its job.  Let's ask @al-caughey .  He may want to to generate a log report to see what's going on.

**Comment by lmuch2020** [comment]
There are others that reported the same issue.  Sherry.....  Oh well.....

**Comment by Eagle-Man** [comment]
What I meant was that the other ones seem to have been fixed with the 1.0.4 update.  Yours is the only one that seems to remain.
